### PR TITLE
DEV-3881-docs: Add readme for sumthemall subject

### PR DESCRIPTION
**Pull request by eslopfer**
Add readme for sumthemall subject

### CODE REVIEW

**Review by Hamzaelkhatri — COMMENTED** [review]
can you add, if the argument is not a number display `0` or `"Error\n"`

**Comment by eslopfer** [comment]
> can you add, if the argument is not a number display `0` or `"Error\n"`

Wouldn't this be one of the unrealistic scenarios, since we are telling the student that the program is going to be tested with numbers, and we are the ones writing the tests?

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/sumthemall/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## sumthemall
+
+### Instructions
+
+Create a program that receives a number of arguments representing whole numbers and displays the sum of the total. If there are not enough arguments or there is an overflow display 0 followed by a new line. If one of the arguments is not a number display 0 followed by a new line.
```
Comment (MSilva95) [inline subjects/sumthemall/README.md:5]: Do you think its good like this?

```
Create a program that takes integers as arguments and displays the sum total of all of them. If it is not a number, there are not enough arguments, or if there is an overflow, print `0` followed by a newline.
```

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/sumthemall/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/sumthemall/README.md]: It is fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

**Pull request merged**